transaction-ballerina/transaction_block.bal

```diff
@@ -105,7 +105,9 @@ function abortTransaction(string transactionId, string transactionBlockId) retur
 transactional function endTransaction(string transactionId, string transactionBlockId)
         returns @tainted string|lang_trx:Error? {
     if (lang_trx:getRollbackOnly()) {
-        return getRollbackOnlyError();
+        lang_trx:Error? e = getRollbackOnlyError();
```

Comment:
Shall we send the same fix to other branches as well?

Reply:
Added a new PR https://github.com/ballerina-platform/module-ballerinai-transaction/pull/439